### ISSUE TITLE
Cannot find name 'GeolocationPositionError' in @types/ol

### ISSUE THREAD

**Issue by tomgruszowski** [hanreev/types-ol#22]
Using Angular 10.1, ol 6.4.3 (same as before update), updated @types/ol from 6.3.x to 6.4.1, using TypeScript 4.0.2

`
ERROR in node_modules/@types/ol/Geolocation.d.ts:77:24 - error TS2304: Cannot find name 'GeolocationPositionError'.
77 constructor(error: GeolocationPositionError);
`

tsconfig.json has:
`"target": "es2015", "module": "es2020", "lib": ["es2018", "dom"]`

Had to add a dummy typing for GeolocationPositionError to suppress error.

**Comment by Charl20cas** [comment]
i have the same error

**Comment by Seairth** [comment]
This appears to be an interface defined by the geolocation API: https://developer.mozilla.org/en-US/docs/Web/API/GeolocationPositionError

I added the following definition just before the `GeolocationError`:

```typescript
export interface GeolocationPositionError {
    code: number;
    message: string;
}
```

I don't see any contribution instructions, so not sure how to get this updated.

**Comment by Seairth** [comment]
Ok.  So, I see that it was [already added in the last commit](https://github.com/hanreev/types-ol/blob/71934c4d269154f65c57f80f84e1aed2b8da3227/%40types/ol/Geolocation.d.ts#L9).  I guess this somehow wasn't included in the npm package that was published a few days later?

**Comment by eacaps** [comment]
As a temporary solution, I was able to temporarily get around this by modifying my tsconfig.json to include: `"typeRoots": ["types","node_modules/@types"]` and in the file `types/ol/index.d.ts` having the following:
```
/**
 * TODO: GeolocationPositionError is missing in @types/ol@6.4.1,
 * if this is fixed then typeRoots in tsconfig.json can be removed
 */
interface GeolocationPositionError {
	code: number;
	message: string;
}

import 'ol';

```

**Comment by hanreev** [comment]
already fixed in 3cdf97c11c5267790faddc5c33409335ce9cd770
